e2e/cloudfront/copilot/frontend/manifest.yml

```diff
@@ -0,0 +1,16 @@
+# We're creating the manifest ahead of time as we want to test the service with a count > 1.
+name: frontend
+type: Load Balanced Web Service
+image:
+  build: 
+    dockerfile: src/Dockerfile
+    context: src
+  port: 80
+
+http:
+  path: '/'
+  alias: ["cloudfront.copilot-e2e-tests.ecs.aws.dev"]
```

Comment:
Does copilot creates A record for this alias when we deploy the service?

Reply:
Yes it does.